swagger/forward/swagger.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
+info:
+  title: Mailgun Relayery Forward Server API
+  description: |
+    API for the server listening to the remote instances' push of messages and forwarding them to MailGun.
+
+    The instances need to provide an authentication token upon pushing to this server.
+  version: "1.0.0"
+host: mailgun.relayery.forward.parquery.com
```

Comment:
I'd remove the host since there is no need to expose parquery-internal setting to the outside world.